google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcStorageOptions.java

```diff
@@ -600,7 +600,12 @@ public GrpcStorageOptions.Builder setBlobWriteSessionConfig(
     @BetaApi
     @Override
     public GrpcStorageOptions build() {
-      return new GrpcStorageOptions(this, defaults());
+      GrpcStorageOptions options = new GrpcStorageOptions(this, defaults());
+      if (options.getUniverseDomain() != null) {
```

Comment:
Can we add a todo-link to the issue to be fixed in sdk-platform-java? Merge away afterward.